lib/Doctrine/ORM/Mapping/ChainTypedFieldMapper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Mapping;
+
+final class ChainTypedFieldMapper implements TypedFieldMapper
+{
+    /**
+     * @readonly
+     * @var TypedFieldMapper[] $typedFieldMappers
+     */
+    private array $typedFieldMappers;
+
+    public function __construct(TypedFieldMapper ...$typedFieldMappers)
+    {
+        $this->typedFieldMappers = $typedFieldMappers;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validateAndCompleteTypedFieldMapping($mapping, $field): array
+    {
+        foreach ($this->typedFieldMappers as $typedFieldMapper) {
+            $mapping = $typedFieldMapper->validateAndCompleteTypedFieldMapping($mapping, $field);
+        }
+
+        return $mapping;
+    }
+}
```

Comment:
Please add a test for this class.

Reply:
There is a test for this class already in `tests/Doctrine/Tests/ORM/Mapping/TypedFieldMapperTest.php` I am not sure why codecov is picking it up as not tested...

Reply:
ah, is it possible that codecov uses PHP < 7.4? As the tests are for PHP 7.4+ (doesnt have sense to test this with PHP version not supporting typed properties)

Reply:
Ive confirmed in debugger the test is being run on PHP7.4+

Reply:
Codecov is supposed to be used on every job. As you can see, it's no longer complaining for some reason :thinking:

Reply:
it is still

Reply:
@derrabus any idea why codecov is complaining here? The test is present, it is executed, the code had to run

Reply:
well now its complaining only about lines with brackets.. I dont get it. If there is anything I can do to improve this, let me know. But the ChainTypedFieldMapper is tested from 2 places - TypedFieldMapperTest and ClassMetadataTest.

Reply:
Indeed, it's just the brackets, so no big deal.